src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -76,7 +77,7 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     UserStore mockUserStore = Mockito.mock(UserStore.class);
     Mockito.when(mockUserStore.isUserRegistered("existing username")).thenReturn(true);
     Mockito.verify(mockRequest).setAttribute("error", "That username is already taken.");
```

Comment:
The order of test statements should be
1. Mockito setup
2. function call: registerServlet.doPost(mockRequest, mockResponse);
3. Verify statements
Refer to the LoginServletTest and fix the order of statements here